File: java/bypstest-api/src/com/wilutions/byps/test/api/ver/EvolveIF.java
package com.wilutions.byps.test.api.ver;

import com.wilutions.byps.BRemote;
import com.wilutions.byps.RemoteException;

/**
 */
public interface EvolveIF extends BRemote {

	public void setEvolve(Evolve obj) throws RemoteException;
	
	public Evolve getEvolve() throws RemoteException;
	
	public void setClient(EvolveIF partner) throws RemoteException;
	
	public EvolveIF getClient() throws RemoteException;
	
	public void sendEvolveToClient() throws RemoteException;
}

File: csharp/bypstest-ser/src-ser/com/wilutions/byps/test/api/ver/EvolveIF.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using com.wilutions.byps;

namespace com.wilutions.byps.test.api.ver
{
	
	// checkpoint com.wilutions.byps.gen.cs.GenRemoteClass:93
	public interface EvolveIF : BRemote {
	
		void SetEvolve(Evolve obj);
		void SetEvolve(Evolve obj, BAsyncResult<Object> asyncResult) ;
		Task SetEvolveAsync(Evolve obj);
		
		Evolve GetEvolve();
		void GetEvolve(BAsyncResult<Evolve> asyncResult) ;
		Task<Evolve> GetEvolveAsync();
		
		void SetClient(EvolveIF partner);
		void SetClient(EvolveIF partner, BAsyncResult<Object> asyncResult) ;
		Task SetClientAsync(EvolveIF partner);
		
		EvolveIF GetClient();
		void GetClient(BAsyncResult<EvolveIF> asyncResult) ;
		Task<EvolveIF> GetClientAsync();
		
		void SendEvolveToClient();
		void SendEvolveToClient(BAsyncResult<Object> asyncResult) ;
		Task SendEvolveToClientAsync();
		
		
	}
}  // end namespace
